main.tsp

```diff
@@ -4,71 +4,76 @@ import "@typespec/versioning";
 using Http;
 using Versioning;
 
-@service(#{ title: "Pet Store", version: "1.0.0" })
+@service(#{ title: "Clustron API", version: "1.0.0" })
 @server("https://example.com", "Single server endpoint")
 @versioned(Versions)
-namespace PetStore;
+namespace Clustron;
 
 enum Versions {
   v1: "1.0.0",
 }
 
-@doc("Represent a Pet available in the PetStore")
-model Pet {
-  id: int32;
+@doc("User settings that are needed for access to the machine.")
```

Comment:
Limiting UserSettings here to “settings needed for access to the machine” feels a bit narrow. If we later want to include things like notification preferences or other settings, this description may quickly become outdated.